deprecated/fabric-containers-v0/src/client/java/net/fabricmc/fabric/impl/client/container/ScreenProviderRegistryImpl.java

```diff
@@ -63,14 +63,14 @@ public <C extends ScreenHandler> void registerFactory(Identifier identifier, Con
 	}
 
 	public static void init() {
-		ClientPlayNetworking.registerGlobalReceiver(ContainerProviderImpl.OPEN_CONTAINER, (client, handler, buf, responseSender) -> {
+		ClientPlayNetworking.registerGlobalReceiver(ContainerProviderImpl.OPEN_CONTAINER, (buf, responseSender, runner) -> {
 			Identifier identifier = buf.readIdentifier();
 			int syncId = buf.readUnsignedByte();
 
 			// Retain the buf since we must open the screen handler with it's extra modded data on the client thread
 			buf.retain();
 
-			client.execute(() -> {
+			runner.run((client) -> {
```

Comment:
Im not sure `runner.run` is ideal. I think it should be made more clear what it really does. context.schedule or something?

Reply:
Yeah we can change it. Other suggestions?